gui/themes/scummclassic/classic_gfx.stx

```diff
@@ -1410,51 +1410,24 @@
 					fg_color = 'lightgrey'
 		/>
 	</drawdata>
-	<drawdata id = 'checkbox_disabled_selected_rtl' cache = 'false'>
```

Comment:
the main reason for removing this and radio buttons from theme files is because the only change they have from the default ones is the change of horizontal align = 'right'
The new abstraction mechanism via code is a better way to approach this problem.

The change of drawing (text first, buttons afterwards), I had implemented via code, so these pieces of drawsteps are no more useful.